Included share-networks to collected stats

jmagudo opened the pull request:
I included shared-networks to collected stats, helps a lot with shared-network with different subnets or ranges.

jmagudo committed:
Added shared networks

jmagudo committed:
Readme changed to reflect sample configuration changes

jmagudo committed:
Modified version information and removed dhcpd-pools binary

jskarpe reviewed (changes requested):
Thank you for the PR!

Your first commit is excellent, however - I don't agree with the other two. I understand the desire to keep the binary out of the repository, however it adds complexity to the installation. This complexity isn't covered in the readme at all now.

If the first commit was in a merge request by itself, I'd very much like to merge it

jmagudo committed:
Undo Modified version information and removed dhcpd-pools binary

jmagudo committed:
Merge branch 'dev' of https://github.com/jmagudo/collectd-iscdhcp into dev

jmagudo commented:
I removed the other commits. I installed the dhcpd-pools package with the following command in Ubuntu:

sudo apt install dhcpd-pools 

(Included in the universe repository)

Because of that I removed it from the repository.
